example_configs/thelounge.md

```diff
@@ -2,15 +2,39 @@
 
 [Main documentation here.](https://thelounge.chat/docs/configuration#ldap-support)
 
-For simple user auth with LLDAP on localhost adapt this in the main config.js:
+Simple Config:
 
 ```
-ldap: {
-  enable: true,
-  url: "ldap://127.0.0.1:3890",
-  tlsOptions: {},
-  primaryKey: "uid",
-  baseDN : "ou=people,dc=example,dc=com",
+      ldap: {
+        enable: true,
+        url: "ldap://localhost:389",
+        tlsOptions: {},
+        primaryKey: "uid",
+        baseDN: "ou=people,dc=example,dc=com",
+    },
 ```
 
-And comment out with ```//``` the entire ```searchDN``` block.
+In this config, The Lounge will use the credentials provided in web ui to authenticate with lldap. It'll allow access if authentication was successful and the user is a member of Base DN.
```

Comment:
I would put the description first, before the config.

Also, all users are members of the base DN, so we don't need to specify that.